File: Kattis/Python/IntroProblems/qaly.py
""" 
kattis qaly problem
Author: Agis Daniels
Solve summation of quality[i]*years[i]
"""
from sys import stdin as rf
def main():
    n=int(rf.readline().strip())
    ans=0
    for line in rf:
        q,y=map(float, line.split())
        ans+=q*y
    print("{:.3f}".format(ans))
if __name__=='__main__':
    main()

File: Kattis/CPP/IntroProblems/qaly.cc
/*
kattis qaly problem
Author: Agis Daniels
Solve summation of quality[i]*years[i]
NOTES
*/
#include <bits/stdc++.h>

using namespace std;

int main (){
    int n=0;
    double q,y,ans=0;
    cin>>n;
    while(n--){
        cin>>q>>y;
        ans+=q*y;
    }   
    cout<<fixed;
    cout<<setprecision(3);
    cout<<ans<<endl;
    return 0;
}
